### ISSUE TITLE
CNN model support

### ISSUE THREAD

**Issue by Saran-nns** [Saran-nns/gradients#2]
Develop and test an example of CNN models with custom kernels and `nn.Conv1D` and `nn.Conv2d` kernels
